cumulusci/tasks/salesforce/tests/test_UpdateDependencies.py

```diff
@@ -144,3 +134,37 @@ def test_update_dependency_latest_option_err(self):
 
         with self.assertRaises(TaskOptionsError):
             task()
+
+    def test_run_task__metadata_bundle(self):
+        project_config = create_project_config()
+        project_config.get_github_api = mock.Mock()
+        task = create_task(
+            UpdateDependencies,
+            {
+                "dependencies": [
+                    {
+                        "repo_owner": "SFDO-Tooling",
+                        "repo_name": "CumulusCI-Test",
+                        "ref": "abcdef",
+                        "subfolder": "src",
+                        "namespace_tokenize": "ns",
+                    }
+                ]
+            },
+            project_config=project_config,
+        )
+        zf = zipfile.ZipFile(io.BytesIO(), "w")
+        task._download_extract_github = mock.Mock(return_value=zf)
+        api = mock.Mock()
+        task.api_class = mock.Mock(return_value=api)
+        task()
+        assert task.install_queue == [
```

Comment:
this is a unittest.TestCase, so you should use self.assertEquals

Reply:
But we're running it with pytest... eh, I'll change it.